pkg/kubernetes/provider_acm_hub.go

```diff
@@ -107,6 +105,9 @@ type acmHubClusterProvider struct {
 	clusterProxyCAFile string
 	watchKubeConfig    bool // whether or not the kubeconfig should be watched for changes
 
+	kubeConfigWatcher *watcher.Kubeconfig
+	clusterWatcher    *watcher.ClusterState
+
```

Comment:
The `kubeConfigWatcher` and `clusterWatcher` fields are declared but never initialized,
causing a panic when `WatchTargets()` or `Close()` is called

see example in provider_single.go:68-70 - Initializes in reset()